hack/olm-artifacts-template.yaml

```diff
@@ -66,6 +69,8 @@ objects:
       env:
         - name: FEDRAMP
           value: "${FEDRAMP}"
+        - name: GOALERT_ENDPOINT_URL
```

Comment:
Nice!, didnt even put two and two together with the FEDRAMP one thats been there. Very handy!